Makefile

```diff
@@ -77,7 +77,7 @@ generate: ## - Generate schema models
 	@printf "${CMD_COLOR_ON} Installing module for go generate\n${CMD_COLOR_OFF}"
 	env GOBIN=${GOBIN} go install github.com/elastic/go-json-schema-generate/cmd/schema-generate@ec19b88f6b5ef7825a928df8274a99337b855d1f
 	@printf "${CMD_COLOR_ON} Installing module for oapi-codegen\n${CMD_COLOR_OFF}"
-	env GOBIN=${GOBIN} go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@latest
+	env GOBIN=${GOBIN} go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@v1.12.4
```

Comment:
@michel-laterman I had to set a fixed version of the openapi code generator to avoid errors during "make check" in CI. it's only a quickfix for the 8.8.1 release but maybe you want to consider fixing it (in the same or a different way) also on the main branch